core-common/build.gradle

```diff
@@ -97,6 +97,8 @@ dependencies {
     compile "org.apache.avro:avro:1.10.2"
     compile "com.fasterxml.jackson.dataformat:jackson-dataformat-avro:2.8.5"
 
+    compile 'com.github.luben:zstd-jni:1.5.0-4'
```

Comment:
Is it the same dependency that the kafka client brings with it or a different one?

Reply:
yes